### ISSUE TITLE
`jax-moseq` produces an error on import because chex needs to be updated.

### ISSUE THREAD

**Issue by wingillis** [dattalab/jax-moseq#32]
A fresh install of `jax-moseq` installs `jax==0.4.20`, which deprecates some classes that `chex==0.1.6` depends on. The solution is to update the version of `chex` to the latest (`0.1.84`).

**Comment by calebweinreb** [comment]
So are you saying that `chex` should be pinned to 0.1.84?

**Comment by wingillis** [comment]
Yes

**Comment by calebweinreb** [comment]
I'm a little worried about this given that keypoint-moseq install currently works for most people, and upgrading chex/jax might open a whole can of worms. When you say "a fresh install of jax-moseq installs jax==0.4.20", what do you mean exactly? Like if you update jax-moseq it tries to update jax too?

**Comment by bainro** [comment]
https://github.com/google-deepmind/chex/pull/251
https://github.com/google-deepmind/chex/tags

Merged to chex master in march, so probably 0.1.8.

I'm having the exact same issue on Ubuntu 22 with GPU. Attemping a newer chex install gives dependency error from jax-moseq...

**Comment by calebweinreb** [comment]
Is the dependency error in pip? If so, I think it can just be ignored.

**Comment by bainro** [comment]
Yeah, just edited my comment as you replied. I did ignore it, thanks Caleb :)
